Makefile

```diff
@@ -33,20 +33,16 @@ docs: .venv/bin/python
 	.venv/bin/mkdocs build
 .PHONY: docs
 
-install-zeus-cli:
-	npm install -g @zeus-ci/cli
-.PHONY: install-zeus-cli
+docserver: .venv/bin/python
+	.venv/bin/pip install -U mkdocs
+	.venv/bin/mkdocs serve
+.PHONY: doc
 
-travis-upload-docs: docs install-zeus-cli
+docs-upload: docs
 	cd site && zip -r gh-pages .
 	zeus upload -t "application/zip+docs" site/gh-pages.zip
 .PHONY: travis-upload-docs
```

Comment:
This target is named the same in Python SDK. If you want it still changed please also change the `.PHONY` line though.

Reply:
Right, let's revert this change then.